README.md

```diff
@@ -1,19 +1,52 @@
-# CodeLingo.io - Code Quality that Scales
 
-## Getting started
+<h3 align="center"> codelingo </h3>
 
-### Quickstart
+<p align="center">
+  <b> Automate Your Reviews on GitHub Pull Requests! </b>
+</p>
+
+<p align="center">
+  <a href="https://github.com/apps/codelingo" target="_blank">
+    <img width="295" height="38" src="https://raw.githubusercontent.com/codelingo/codelingo/master/public/img/install.png" />
+  </a>
+</p>
+
+# Lingo
+
+Lingo is a CLI tool to run [CodeLingo](https://www.codelingo.io) with any git repository on your local machine.
+
+### Quick Start
+
+In this quick start we'll review a Golang git repository on your local machine for common Go issues.
+
+After installing the [lingo tool](https://github.com/codelingo/lingo/releases/latest), set it up with the following commands:
 
 ```bash
-# Run this command from anywhere. Follow the prompts to set up CodeLingo on your machine.
+# Run this command from anywhere. Follow the prompts to set up Codelingo on your machine.
 $ lingo config setup
 
-# Run this command to add a .lingo file in the current directory.
+# Run this command inside a git repository to add a default .lingo.yaml file in the current directory.
 $ lingo init
+```
 
-# Add a Tenet to the .lingo file (see https://codelingo.io/docs/concepts/tenets/#writing-custom-tenets for more info). This will be used by the following command to run a review.
-$ lingo review -i
+Replace the default content of the .lingo.yaml file we generated above with:
 
+```yaml
+  tenets:
+    - import: codelingo/go
 ```
 
-See [https://www.codelingo.io/docs/#getting-started](https://www.codelingo.io/docs/#getting-started) for more detailed instruction
\ No newline at end of file
+Run the Review Flow to check your source code against the go Tenet bundle we imported above:
+
+```bash
+# Run this command from the same directory as the .lingo.yaml file or any of its sub directories.
+$ lingo run review
+```
+
+<!-- TODO add screenshot of lingo review -->
+
+## Slow Start
+
+Follow the [step by step guide](https://www.codelingo.io/docs/getting-started) to using lingo.
```

Comment:
Incorrect link, should be: https://www.codelingo.io/docs/#getting-started

Reply:
this is actually the correct link: https://www.codelingo.io/docs/getting-started/ which is the intro to lingo CLI. If the "/" is left off, it redirects to https://www.codelingo.io/getting-started/.